cinema/models.py

```diff
@@ -35,12 +39,24 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    _, extension = os.path.splitext(filename)
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}.{extension}"
+
+    return os.path.join("uploads/movies", filename)
```

Comment:
```suggestion
    return os.path.join("uploads", "movies", filename)
```
Better to do like this. It will works in all OS